Makefile

```diff
@@ -1,11 +1,28 @@
-all: index.html
+SHELL=/bin/bash
 
-force:
-	bikeshed -f spec ./index.src.html
+.PHONY: ci clean local remote
 
-index.html: index.src.html
-	bikeshed -f spec ./index.src.html
+local: index.bs
+	bikeshed --die-on=warning spec index.bs index.html
 
-publish:
-	git push origin master master:gh-pages
+remote: index.html
 
+ci: index.bs index.html
+	mkdir -p out
+	cp index.html out/
+
+clean:
+	rm index.html
+
+index.html: index.bs
+	@ (HTTP_STATUS=$$(curl https://api.csswg.org/bikeshed/ \
+	                       --output index.html \
+	                       --write-out "%{http_code}" \
+	                       --header "Accept: text/plain, text/html" \
+												 -F die-on=warning \
```

Comment:
This spec probably hasn't been building with "die-on=warning" so far so you might want to omit this (both here and on line 6) until it's building cleanly without warnings.

Actually it looks like the spec is building with `-f` (`--force`) which causes certain errors to be ignored. So you might need to do that too. (It'd be `-F force=1` for this line.)

Reply:
As discussed on chat, die-on=warning works here which is great, so we'll stick with this.